components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/AbstractApplicationAuthenticator.java

```diff
@@ -386,4 +386,17 @@ public String getAuthMechanism() {
         diagnosticLog.info("AuthMechanism is set to : " + authMechanism + " for the authenticator: " + getName());
         return authMechanism;
     }
+
+    @Override
+    public String[] getTags() {
+
+        String tags = getAuthenticatorConfig().getParameterMap().get(FrameworkConstants.TAGS);
+        String[] tagList;
+        if (StringUtils.isEmpty(tags)) {
```

Comment:
we can use StringUtils.split which handles the empty case

Reply:
Addressed with commit ad110cdff4ffcfe5766b31f1a09bebaf633730e6